hvplot/converter.py

```diff
@@ -334,6 +334,24 @@ class HoloViewsConverter:
     -------------
     bgcolor (default=None): str
         Background color of the data area of the plot
+    color: str or list, optional
+        Color(s) to use for the plot. Accepts a single color, a list of colors, or
+        a column name from the data to map colors based on values.
+    c: str or list, optional
+        Alias for `color`. Used for specifying colors in scatter plots and other
+        elements that support color mapping.
+    cmap: str or list, optional
+        The colormap to use for continuous or categorical color mapping.
+        Accepts colormap names, dictionaries for discrete mapping,
```

Comment:
- You write that it supports various types but that isn't reflected in the docstring types (`str or list`).
- As a user, how can I find the colormap names that I could set?
- What are the keys and values I should use to build the dictionary?
- What colormap objects can I use?
- What values can the custom list contain?